feature-libs/product/configurators/common/src/components/config-form/config-form.component.ts

```diff
@@ -17,8 +19,28 @@ import { ConfigFormUpdateEvent } from './config-form.event';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ConfigFormComponent implements OnInit {
-  configuration$: Observable<Configurator.Configuration>;
-  currentGroup$: Observable<Configurator.Group>;
+  configuration$: Observable<
+    Configurator.Configuration
+  > = this.configRouterExtractorService.extractRouterData().pipe(
+    filter(
+      (routerData) =>
+        routerData.pageType === ConfigurationRouter.PageType.CONFIGURATION
+    ),
+    switchMap((routerData) => {
+      return this.configuratorCommonsService.getOrCreateConfiguration(
+        routerData.owner
+      );
+    })
+  );
+  currentGroup$: Observable<
+    Configurator.Group
+  > = this.configRouterExtractorService
+    .extractRouterData()
+    .pipe(
+      switchMap((routerData) =>
+        this.configuratorGroupsService.getCurrentGroup(routerData.owner)
+      )
+    );
 
   UiType = Configurator.UiType;
```

Comment:
Let's follow the convention, and make this one to start with a lowercase `u`.

Reply:
In this case it's more to expose the enum to the UI..shouldn't we keep it as it is?
(Otherwise we need 9 public methods to check for the different types)

Reply:
I think we misunderstood each other :)
I was suggesting to do this:
```suggestion
  uiType = Configurator.UiType;
```
rather than to have it as `UiType = Configurator.UiType;` (notice the capital `U` as the property name).